api-extractor/src/DocItemLoader.ts

```diff
@@ -132,8 +140,8 @@ export default class DocItemLoader {
       }
     );
 
-    // JSON schema has been found and validated
-    this.cache[cachePackageName] = apiPackage;
+    const packageName: string = path.basename(packageJsonFilePath).split('.').shift();
```

Comment:
I would have to do: path.parse(packageJsonFilePath).name.split('.').shift(), since if the filename is es6-collections.api.json then path.parse().name gives 'es6-collections.api'. So this is essentially the same thing.